src/helpers/MobileSafariScroll.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
```

Comment:
I miss some documentation here, because without context the purpose of this component is unclear.
I think a one-sentence summary + link to GitHub/Slack conversations about this should be enough

Reply:
fixed

Reply:
I see it now! Thanks